cea/plots/optimization/annual_costs.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import division
+from __future__ import print_function
+
+import plotly.graph_objs as go
+
+import cea.plots.optimization
+from cea.plots.variable_naming import NAMING, COLOR
+
+__author__ = "Daren Thomas"
+__copyright__ = "Copyright 2019, Architecture and Building Systems - ETH Zurich"
+__credits__ = ["Jimeno A. Fonseca", "Daren Thomas"]
+__license__ = "MIT"
+__version__ = "0.1"
+__maintainer__ = "Daren Thomas"
+__email__ = "cea@arch.ethz.ch"
+__status__ = "Production"
+
+
+class AnnualCostsPlot(cea.plots.optimization.GenerationPlotBase):
+    """Implement the "CAPEX vs. OPEX of centralized system in generation X" plot"""
+    name = "Bar-chart of costs"
```

Comment:
Why do `name` and `title` differ so much? I think that makes it harder for the user to find the plot again...

Reply:
I can change this...